packages/dashboard/src/components/SubwayNav/SubwayNav.tsx

```diff
@@ -0,0 +1,289 @@
+import * as React from 'react';
+
+import { classNamesFunction, css } from 'office-ui-fabric-react/lib/Utilities';
+import { ISubwayNavProps, ISubwayNavStep, ISubwayNavStyles, SubwayNavStepState } from './SubwayNav.types';
+import { Icon, Link } from 'office-ui-fabric-react';
+import { getSubwayNavStyles } from './SubwayNav.styles';
+
+export interface IIconProps {
+  iconName: string;
+
+  iconClassName: string;
+}
+
+const getClassNames = classNamesFunction<ISubwayNavProps, ISubwayNavStyles>();
+const classNames = getClassNames(getSubwayNavStyles!);
+
+/**
+ * Component for Subway control
+ */
+export class SubwayNav extends React.Component<ISubwayNavProps, {}> {
+  constructor(props: ISubwayNavProps) {
+    super(props);
+  }
+
+  /**
+   * Render the Subway Nav control
+   */
+  public render(): JSX.Element {
+    const { steps /*theme, */ } = this.props;
+    let stepElements: JSX.Element[] = [];
+    let renderedSteps: JSX.Element[] = [];
+
+    steps.map((step: ISubwayNavStep, index: number) => {
+      renderedSteps = this._renderStep(step, index === 0 ? undefined : steps[index]);
+      stepElements = stepElements.concat(renderedSteps);
+    });
+
+    return (
+      <div className={css(this.props.className, classNames.subwayNavContainer)}>
+        <div className={classNames.subwayNavContentContainer}>
+          <div className={classNames.subwayNavContent}>{stepElements}</div>
+        </div>
+      </div>
+    );
+  }
+
+  /**
+   * Render step of the subway Nav control
+   * @param stepToRender
+   * @param prevStep
+   */
+  private _renderStep(stepToRender: ISubwayNavStep, prevStep: ISubwayNavStep | undefined): JSX.Element[] {
+    let stepElements: JSX.Element[] = [];
+
+    const iconProps = this._getIconPropsStep(stepToRender);
+
+    if (prevStep !== undefined) {
+      const connectorClass = this._getStepConnectorClassName(stepToRender);
+      stepElements.push(
+        <div className={css(classNames.subwayNavStepConnector, connectorClass)} key={'connector' + '-' + stepToRender.key} />
+      );
+    }
+
+    const renderSubStep = stepToRender.state === SubwayNavStepState.Current && this._hasSubSteps(stepToRender);
+
+    stepElements.push(
+      <div
+        key={stepToRender.key}
+        className={css(
+          classNames.subwayNavStepDiv,
+          stepToRender.state !== SubwayNavStepState.Current && stepToRender.disabled ? classNames.disableStep : undefined
+        )}
+      >
+        <Icon iconName={iconProps.iconName} className={css(iconProps.iconClassName, classNames.subwayNavStepIcon)} />
+        <Link
+          // tslint:disable-next-line jsx-no-lambda
+          onClick={() => {
```

Comment:
This will kill perf. We shouldn't disable linting like this unless there's no recourse. We also don't need to use a link here I think it adds additional complexity. We'll also want this to be a different component for portability. We can reconcile changes once this is in but wanted to mention the perf issues here.

Reply:
Does the tab order work properly here onClick for accessibility?

---
In reply to: [253280655](https://github.com/OfficeDev/office-ui-fabric-react/pull/7812#discussion_r253280655) [](ancestors = 253280655)

Reply:
Added ariaLabel for Link

Reply:
I will test accessibility integrated with wizard.  Also, I know Brandon also has some code for this.